### DISCUSSION TITLE
Steam auth

### DISCUSSION THREAD

**Discussion by modyabhi** [nextauthjs/next-auth#5413] [Ideas]
### Question 💬

There's lack of documentation around support for steams auth( open id 2.0), if there is can someone show help me with the usage?

### How to reproduce ☕️

 Not sure

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
Hi, we don't currently have a built-in provider for Steam, but we would accept a PR! :green_heart:

Some useful links:

Built-in providers: https://github.com/nextauthjs/next-auth/tree/main/packages/next-auth/src/providers
Build-in providers' documentation: https://github.com/nextauthjs/next-auth/tree/main/docs/docs/providers

OAuth provider configuration: https://next-auth.js.org/configuration/providers/oauth

**Reply by modyabhi** [reply]
Sorry I accidentally clicked the PR button I'm not that much experienced in authentication, hence I've turned to libraries like this to make authentication easier. 

I feel like there's enough demand for a solution as there are millions of users using steam and want solution that are fast like nextjs. 

Any help would be greatly appreciated.

**Reply by 1337cookie** [reply]
Please see my comment in this other discussion here: https://github.com/nextauthjs/next-auth/discussions/697?sort=old#discussioncomment-4392541
